tests/run_tests.sh

```diff
@@ -18,16 +19,30 @@ function before_run() {
 }
 
 function before_test() {
+   # TODO:
+   # It might be faster to rollback to a snapshot taken before a run here
+   # since updating apt cache can take a bit of time.
+   # Unfortunately, it would require users installing more plugins or
+   # updating vagrant. Snapshots are built into vagrant 1.8.0:
```

Comment:
1.8.0 is over a year old, and upgrading Vagrant doesn't invalidate existing instances. I don't see the value in significantly-downrev support here.